setup.py

```diff
@@ -1,19 +1,21 @@
 #!/usr/bin/env python
 
 import os
+import setuptools  # noqa
 
-import setuptools
 from distutils.core import setup
 
-execfile('grouper/version.py')
+from grouper.version import __version__
```

Comment:
This explicitly uses execfile so that grouper doesn't have to be importable and on the path in order to build grouper.